### ISSUE TITLE
Add 'clientAuthenticatorType' to the 'updateClient' migration

### ISSUE THREAD

**Issue by IngoStrauch2020** [mayope/keycloakmigration#83]
Hi there,

I'm currently experimenting with replacing the usage of a client-secret with a client-jwt plus JWKS URL for identity brokering between two test realms, so that the client JSON in "my-test-realm" will look like this:
```
	"clientAuthenticatorType": "client-jwt",
	"attributes": {
		"use.jwks.url": "true",
		"jwks.url": "http://localhost:9094/realms/my-other-test-realm/protocol/openid-connect/certs"
    }
```
That can be specified in the JSON to be imported for new clients, but when I want to migrate an existing client, it's currently not possible to set the "clientAuthenticatorType".

I guess I'll be able to implement this myself, but it's not super-urgent, so it might take a while.

**Comment by klg71** [comment]
Hey @IngoStrauch2020 :) Thanks for opening the issue! If you need any help let me know :)

**Comment by klg71** [comment]
Released 0.2.69. Could you retest it?

**Comment by IngoStrauch2020** [comment]
New version of the plugin works with our previous migrations plus a new one that sets the clientAuthenticatorType.
Thanks for the quick release!
